pkg/psmdb/service.go

```diff
@@ -118,7 +123,12 @@ func ExternalService(cr *api.PerconaServerMongoDB, replset *api.ReplsetSpec, pod
 	default:
 		svc.Spec.Type = corev1.ServiceTypeClusterIP
 	}
-
+	if cr.CompareVersion("1.16.0") > 0 {
```

Comment:
with this we won't have this field on `v1.16.0` please also include 1.16.0
```suggestion
	if cr.CompareVersion("1.16.0") >= 0 {
```